source/install/ee-prod-rhel-7.rst

```diff
@@ -161,7 +161,7 @@ Set up Mattermost Server
 8. Set up Mattermost to use the systemd init daemon which handles
    supervision of the Mattermost process. 
 
-   * Create and edit ``/etc/systemd/system/mattermost.service``
+   * Create and edit ``/usr/lib/systemd/system/mattermost.service``
```

Comment:
The `/usr/lib/systemd/system/` directory is for unit files that are distributed with installed RPM packages, and `/etc/systemd/system/` is for unit files that are created by `systemctl enable`, which includes Mattermost. We should keep `/etc/systemd/system/` as the location in the Mattermost instructions.